src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ObjectWriter.cs

```diff
@@ -372,10 +394,55 @@ public void EmitDebugVarInfo(ObjectNode node)
         }
 
         [DllImport(NativeObjectWriterFileName)]
-        private static extern void EmitDebugFunctionInfo(IntPtr objWriter, byte[] methodName, int methodSize);
-        public void EmitDebugFunctionInfo(int methodSize)
+        private static extern void EmitEHClause(IntPtr objWriter, UInt32 TryOffset, UInt32 TryLength, UInt32 HandlerOffset, UInt32 HandlerLength);
+
+        public void EmitEHClause(EHClauseInfo ehClause)
+        {
+            try
```

Comment:
It would be useful to have all places with workarounds for the out-of-sync object writer package to be marked with a easy to search for TODO, with a link to https://github.com/dotnet/corert/issues/5177

Reply:
done